gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
Hmm! This is very strange. I have touched 3 files only, but it shows that I have updated 13 files. Possible merge issue?